src/client/components/Navigation/Topnav.js

```diff
@@ -980,22 +992,24 @@ class Topnav extends React.Component {
           </div>
           <div className="Topnav__right-bottom">
             {this.content()}
-
-            {isAuthenticated && (
-              <div className="Topnav__broker">
+            <div
+              className={classNames('Topnav__broker', {
+                'justify-end': !isAuthenticated || platformName === 'widgets',
+              })}
+            >
+              {!isAuthenticated || platformName === 'widgets' ? (
```

Comment:
show broker button only for authenticated user

Reply:
+++